.github/workflows/i18n.yml

```diff
@@ -0,0 +1,33 @@
+name: I18n
+
+on:
+  schedule:
+    - cron: '0 6,18 * * *'
+
+jobs:
+  translation-strings:
+    name: Translation strings
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set PHP version
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
+          tools: composer:v2
+
+      - name: Install dependencies
+        run: composer install
+
+      - name: Run translation script
+        run: php ./bin/i18n.php
+
+      - name: Commit and push
+        uses: actions-js/push@v1.2
```

Comment:
Thinking about security a bit more here, I don't think there's any viable threat from the PHP action above, because it wouldn't have any way to make a commit to our repo. It could do weird things with PHP or composer, but nothing would be able to leak outside of the sandbox. This action, on the other hand, gets our `GITHUB_TOKEN` which is what allows it to make commits for us. This is the one we'd most likely want to pin to a specific version.

Reply:
`shivammathur/setup-php@v2` installs PHP though, right? So it could install a malicious version of PHP, and when `php ./bin/i18n.php` is run, the compromised `file_put_contents()` could append `eval( 'evil payload' )` to the end of ` 'translation-strings.php`. Then w.org would `include` the compromised file, and execute the payload on production. Does that seem accurate?

Reply:
> Then w.org would include the compromised file, and execute the payload on production.

This part isn't accurate, because the generated file doesn't exist on production, except during the brief window when the pot file generator [clones this repo](https://dotorg.trac.wordpress.org/browser/wordpress/website/translate/bin/init-repos.sh#L28) in order to statically scan the files for translation strings. The file is never included anywhere during that process and never interpreted as PHP. The rest of the time, the generated file lives in a directory in this repo that is not synced to SVN.

Reply:
Er, I guess once the repo has been checked out for the pot file generation, it doesn't get deleted afterward. But it still doesn't get used for anything else or run through a PHP interpreter.

Reply:
Ah, ok, awesome 👍🏻   Then I agree that need to worry about it.